polyaxon/values.yaml

```diff
@@ -634,3 +634,13 @@ auth: {}
 istio: {}
 operators: {}
 ui: {}
+
+# priority for all pods created by this helm chart
+priorityClassName: ""
+
+# additional labels for all pods created by this helm chart
+podLabels: {}
```

Comment:
I think we should call this `customLabels` and `customAnnotations`.
And use `"config.labels.custom"` and `"config.annotations.custom"`.

Sorry for late review.

Reply:
Actually even better let's just call it `labels` and `annotations`. This should be the global definition, after merging we make another PR to add per deployment customization similar to how we do now for `nodeSelector`, `tolerations` and `affinity`, here's an example of how it will look like after the final change: e.g. [nodeSelector](https://github.com/polyaxon/charts/blob/master/polyaxon/templates/partials/_gateway.tpl#L9)